pyproject.toml

```diff
@@ -26,7 +26,7 @@ dependencies = [
     "ruamel.yaml >=0.16.10",
     "ruamel.yaml.jinja2",
     "setuptools >=30.3.0",
-    "semver~=2.13.0",
```

Comment:
In theory, b/c of this, grayskull should not install semver 3.0.0, right? Anyway, I got it in my env and it broke grayskull. This PR makes it a bit future proof.